templates/init/functions/javascript/index.js

```diff
@@ -1,8 +1,8 @@
 const functions = require('firebase-functions');
 
-// // Create and Deploy Your First Cloud Functions
-// // https://firebase.google.com/docs/functions/write-firebase-functions
-//
+// Create and Deploy Your First Cloud Functions
```

Comment:
I think these extra slashes were here so that you could take the whole text, uncomment, and have it work. I've seen Doug use this in his demos and it saves time. - perhaps we should leave those in.

Reply:
There's code above this that isn't commented out, so "the whole text" is rather "everything from the third line down". I find it weird that we have double comment markers - it's not really a natural thing to do.